ui/desktop/electron-app/src/cli/path.js

```diff
@@ -5,11 +5,16 @@
 
 const path = require('path');
 const { isWindows } = require('../helpers/platform.js');
+const isDev = require('electron-is-dev');
 
 module.exports = {
   // Returns boundary cli path
   path: () => {
     const name = isWindows() ? 'boundary.exe' : 'boundary';
-    return path.resolve(process.resourcesPath, 'cli', name);
+
+    // The CLI will be located in a different location once packaged up in an ASAR
+    return isDev
+      ? path.resolve(__dirname, '..', '..', 'cli', name)
+      : path.resolve(process.resourcesPath, 'cli', name);
```

Comment:
This was a bug I created when fixing the CLI path in production.

I forgot about dev and broke normal development 😓

Reply:
Good catch. This is why I am only seeing the issue when I am in dev.